api/envoy/api/v2/cluster/outlier_detection.proto

```diff
@@ -80,4 +80,30 @@ message OutlierDetection {
   // used to disable ejection or to ramp it up slowly. Defaults to 0.
   google.protobuf.UInt32Value enforcing_consecutive_gateway_failure = 11
       [(validate.rules).uint32.lte = 100];
+
+  // Determines whether to distinguish local origin failures from external errors. If set to true
+  // the following configuration parameters are taken into account:
+  // consecutive_local_origin_failure, consecutive_local_origin_failure
```

Comment:
Please ref link all fields. Same elsewhere in this change.

Reply:
Done. Here and everywhere else. Thanks for catching it.